soccer/gameplay/tactics/our_placement.py

```diff
@@ -59,11 +59,20 @@ def execute_dribble(self):
 
     def on_exit_dribble(self):
         self.robot.set_dribble_speed(0)
-        self.remove_subbehavior('dribble')
+        self.remove_all_subbehaviors()
+        print("EXIT DRIBBLE")
 
     def on_enter_pause(self):
+        self.robot.is_ball_placer = True
         self.pause_time = time.time()
 
+        #without this the robot tries to leave the circle in a random direction
+        self.robot.set_max_speed(0.01)
```

Comment:
This sounds like an issue with the ball obstacle? If so, we should proably remove the obstacle and set the max speed low.